### PR TITLE
Fixed bugs hopefully

### PR DESCRIPTION
**Pull request by sa-ma**

### CODE REVIEW

File: timeline.php [hunk 1]
```diff
@@ -17,19 +29,19 @@
           </div>
 
           <div class="personal-info text-center ">
-            <h1><?php echo $name; ?></h1>
+            <h1>Tamz Hallz</h1>
             <div class="font-weight-bold skillset">
-              <?php echo $email; ?>
+              UI/UX Designer | Content Designer | Product Newbie
```
Comment (MyloCarson) [inline timeline.php]: Why this change?

File: timeline.php [hunk 2]
```diff
@@ -17,19 +29,19 @@
...
             <div class="shortbio">
               I'm kinda weird in a good way..... I design for survival
             </div>
           </div>
 
-          <div class="sidebar-icons text-center my-3">
-            <i class="fab fa-github social"></i>
-            <i class="fab fa-dribbble social"></i>
-            <i class="fas fa-rss social"></i>
+          <div class="sidebar-icons text-center my-3">
```
Comment (MyloCarson) [inline timeline.php]: We agreed no font-awesome icons. Please

File: timeline.php [hunk 3]
```diff
@@ -17,19 +29,19 @@
...
               I'm kinda weird in a good way..... I design for survival
             </div>
           </div>
 
-          <div class="sidebar-icons text-center my-3">
-            <i class="fab fa-github social"></i>
-            <i class="fab fa-dribbble social"></i>
-            <i class="fas fa-rss social"></i>
+          <div class="sidebar-icons text-center my-3"> 
+            <a href="https://github.com"><i class="fab fa-github social"></i></a>
```
Comment (MyloCarson) [inline timeline.php]: We agreed no font-awesome icons naaa

File: timeline.php [hunk 4]
```diff
@@ -354,11 +355,16 @@ class="form-control post-text"
...
+                
             </div>
-
+            
         </div>
 
     </div>
 
-  <?php include 'footer.php'; ?>
+    <script>
```
Comment (MyloCarson) [inline timeline.php]: Why did remove footer.php line?

**Comment by MyloCarson** [comment]
I cnt merge into master. Please create a new PR and use the required branch FE or BE as base

File: timeline.php (hunk 3 above)
Comment (sa-ma) [reply timeline.php]: Who is the we? When was it agreed? And why am I just finding out.

File: timeline.php (hunk 1 above)
Comment (sa-ma) [reply timeline.php]: Not my change I pulled before pushing. I'll close the pr the changes has been made already

File: timeline.php (hunk 3 above)
Comment (MyloCarson) [reply timeline.php]: Err, just make another PR to FE without conflicts. I am open to review

Comment (sa-ma) [reply timeline.php]: I've done that already and it's been merged. Discard this pr. It was a desperate attempt to save time
